### ISSUE TITLE
Supposedly out of memory when managing an output folder with too many files

### ISSUE THREAD

**Issue by ondrejhlavacek** [keboola/docker-bundle#429]
![image](https://user-images.githubusercontent.com/497675/57917246-9746c000-7894-11e9-9ec6-709c41a06fff.png)

![image](https://user-images.githubusercontent.com/497675/57917254-9ada4700-7894-11e9-86a6-7b651c12dd01.png)

![image](https://user-images.githubusercontent.com/497675/57917329-bd6c6000-7894-11e9-8dce-eb351b7026c6.png)

https://connection.keboola.com/admin/projects/258/jobs/505306651

sapi job je založenej 

![image](https://user-images.githubusercontent.com/497675/57917368-cfe69980-7894-11e9-867d-986f20c598bc.png)

muselo to exnout někde po založení jobu

![image](https://user-images.githubusercontent.com/497675/57917550-34095d80-7895-11e9-991a-ffa82d283045.png)

**Comment by Halama** [comment]
Podobná situace ale padlo to ještě před OM. Zřejmě po spuštění merge procesoru? Možná nezafungoval správně.
Měl jsem nastavený limit na 100k souborů v S3 extractoru.

https://connection.keboola.com/admin/projects/258/jobs/505387222

![image](https://user-images.githubusercontent.com/903531/57968829-03e2bd00-7970-11e9-882e-353db0a6b822.png)


![image](https://user-images.githubusercontent.com/903531/57968828-ff1e0900-796f-11e9-8d55-09f898308661.png)

![image](https://user-images.githubusercontent.com/903531/57968859-88354000-7970-11e9-9b7d-f94a7a8ff376.png)

**Comment by ondrejhlavacek** [comment]
jo, to jsem už taky někde zahlídl, to bude taky https://github.com/keboola/docker-bundle/issues/430
